scripts/build-android.js

```diff
@@ -51,24 +51,24 @@ if (options.arch) {
 
 const buildType = options.buildType;
 
-const ndkPath = process.env["ANDROID_NDK"];
+const ndkPath = process.env["ANDROID_NDK"] || process.env["ANDROID_NDK_HOME"];
 if (!ndkPath) {
-  throw Error("ANDROID_NDK environment variable not set");
+  throw Error("ANDROID_NDK / ANDROID_NDK_HOME environment variable not set");
 }
 
 const sdkPath = getAndroidSdkPath();
 const cmakePath = getCmakePath(sdkPath);
 const cmakeVersion = getCmakeVersion(sdkPath);
 
-const buildPath = path.resolve(process.cwd(), "build-realm-android");
+const buildPath = path.resolve(process.cwd(), "prebuilds", "build-realm-android");
```

Comment:
Since `prebuilds` are specifically referring to the artifacts produced by the [prebuild](https://www.npmjs.com/package/prebuild) package, I'd say we should leave the android artifacts outside of that directory.